trustchain_design.md

```diff
@@ -112,25 +112,25 @@ The author also stores the block they just emitted, preventing the *Trustchain*
+When first receiving a *device* creation block for a *user*, the recipient needs to trust this block relates to the correct *user*. The block and the information it contains &mdash; especially the new *device*'s [Device Encryption Key Pair] and [Device Signature Key Pair] &mdash; are then stored locally, preventing the need to ask for this *device*'s public key again.
 
-It is to be noted that because of the signature delegation scheme, the entity that needs to be trusted upon receiving blocks for a new *user* is **not** the *Trustchain* distribution system, but the private [Trustchain Signature Key Pair] holder (i.e. the *application server*).
+It has to be noted that because of the signature delegation scheme, the entity that needs to be trusted upon receiving blocks for a new *user* is **not** the *Trustchain* distribution system, but the private [Trustchain Signature Key Pair] holder (i.e. the *application server*).
 
 ### Forward secrecy
 
-Forward secrecy is, despite being an interesting property, **not **implemented in *Tanker Core*. The reason is that *Tanker Core* is not targeted solely at encrypting ephemeral message exchanges, but also persistent data. We want *users* to be able to decrypt anything at any time with any of their *devices*, which means that we cannot use a forward secure scheme.
+Forward secrecy is, despite being an interesting property, **not** implemented in *Tanker Core*. The reason is that *Tanker Core* is not targeted solely at encrypting ephemeral message exchanges, but also persistent data. We want *users* to be able to decrypt anything at any time with any of their *devices*, which means that we cannot use a forward secure scheme.
 
-We consider implementing an optional forward secure scheme tailored for messaging in a future version of *Tanker Core*.
+We are considering implementing an optional forward secure scheme tailored for messaging in a future version of *Tanker Core*.
 
 ## Separation of trust
 
-*Tanker Core* is based on the separation of trust. Any *user* needs to trust:
+*Tanker Core* is based on the separation of trust principle. Any *user* needs to trust:
 
 - the *application* to store and distribute encrypted data
 - *Tanker* to distribute blocks
 - *Tanker* and the *application* not to collude
 
 A collusion between *Tanker* and the *application* could result in a split view attack, in which an alternate, seemingly valid version of the *Trustchain* is strategically distributed to *users* in order to trick them into sharing data with the wrong entity.
 
-We plan on implementing a gossiping scheme in future versions of *Tanker Core*, which would prevent split view attacks.
+We are considering to implement a gossiping scheme in future versions of *Tanker Core*, which would prevent split view attacks.
```

Comment:
implementing*

https://thegrammarexchange.infopop.cc/topic/consider-to-do-consider-doing

remove this too?

Reply:
yes too